MetaMask/metamask-extension
# MV3: catch "Duplicate script ID 'inpage'" error

## Explanation
Currently, when the `app-init.js` service worker restarts, it attempts to inject another copy of the `inpage.js` script. 

This PR proposes to catch the error for now. Please read comment in code.

## More Information

Fixes https://github.com/MetaMask/metamask-extension/issues/15965

## Screenshots/Screencaps

Note that when the `inpage.js` script is not loaded on time, it will produce this error: 
<img src="https://user-images.githubusercontent.com/20778143/192319721-8bd32ffa-321a-4b7b-83a5-32f8410c6253.png" width="460">

## Manual Testing Steps

1. Launch extension MV3
2. Open dapp
3. Inspect `app-init.js` service worker
4. Terminate `app-init.js` service worker
5. Observe error

## Pre-Merge Checklist

- [ ] ~**IF** this PR fixes a bug, a test that _would have_ caught the bug has been added~ - no tests for chrome extension pages and service worker exist at this time

### + If there are functional changes:
